crates/tor-dirmgr/src/state.rs

```diff
@@ -730,7 +779,7 @@ impl<DM: WriteNetDir> DirState for GetMicrodescsState<DM> {
     fn add_from_cache(
         &mut self,
         docs: HashMap<DocId, DocumentText>,
-        storage: Option<&Mutex<SqliteStore>>,
+        _storage: Option<&Mutex<SqliteStore>>,
```

Comment:
no need to bind it to a value

```suggestion
        _: Option<&Mutex<SqliteStore>>,
```